modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -314,29 +314,41 @@ a|Print help information.
 a|`<device>`
 a|The destination device.
 
-2+|*coreos-installer ISO Ignition subcommands*
+2+|*coreos-installer ISO subcommands*
 
 |*_Subcommand_* |*_Description_*
 
-a|`$ coreos-installer iso ignition embed <options> --ignition-file <file_path> <ISO_image>`
-a|Embed an Ignition config in an ISO image.
+a|`$ coreos-installer iso customize <options> <ISO_image>`
+a|Customize a {op-system} live ISO image.
 
-a|`coreos-installer iso ignition show <options> <ISO_image>`
-|Show the embedded Ignition config from an ISO image.
+a|`coreos-installer iso reset <options> <ISO_image>`
+|Restore a {op-system} live ISO image to default settings.
 
 a|`coreos-installer iso ignition remove <options> <ISO_image>`
 a|Remove the embedded Ignition config from an ISO image.
 
-2+|*coreos-installer ISO Ignition subcommand options*
+2+|*coreos-installer ISO Customize subcommand options*
 
 |*_Option_* |*_Description_*
 
+a|`--dest-ignition <path>`
+a|Merge specified Ignition config into a new config fragment for destination system.
+
+a|`--dest-karg-append <arg>`
+a|Add a kernel argument to boots of the destination system.
```

Comment:
Sounds a bit odd to me, maybe something like this?
```suggestion
a|Add a kernel argument to the destination's system boots.
```

Reply:
I'm just now realizing it sounds like the destination system has feet and wears boots 😄 thanks for catching!